hasher-matcher-actioner/hmalib/lambdas/pdq/pdq_indexer.py

```diff
@@ -67,12 +67,21 @@ def lambda_handler(event, context):
         codecs.getreader("utf-8")(pdq_data_file["Body"]),
         fieldnames=PDQ_DATA_FILE_COLUMNS,
     )
-    pdq_data = [(row["hash"], int(row["id"])) for row in pdq_data_reader]
+    pdq_data = [
+        (
+            row["hash"],
+            # Also add hash to metadata for easy look up on match
```

Comment:
Explain this to me? Why isn't the ID enough for easy lookup on match?

Reply:
See @schatten's note below.